src/aioswitcher/bridge.py

```
@@ -428,10 +435,17 @@ async def start(self) -> None:
             protocol_factory = UdpClientProtocol(
                 partial(_parse_device_from_datagram, self._on_device)
             )
+            # Bind with SO_REUSEADDR so a reload can rebind the port while the
+            # previous socket is still lingering in TIME_WAIT, instead of failing
+            # with "address already in use" (errno 98) until a full restart. This
+            # follows the same pattern aioshelly uses for its CoAP listener.
+            sock = socket(AF_INET, SOCK_DGRAM)
+            sock.setsockopt(SOL_SOCKET, SO_REUSEADDR, 1)
+            sock.setblocking(False)
+            sock.bind(("0.0.0.0", broadcast_port))  # nosec
```

Ruff (0.15.21)

[error] Possible binding to all interfaces

(S104)